package.json

```diff
@@ -94,24 +94,30 @@
     "yup": "0.28.2"
   },
   "devDependencies": {
-    "@babel/core": "7.12.16",
-    "@babel/plugin-proposal-class-properties": "7.12.13",
-    "@babel/plugin-proposal-object-rest-spread": "7.12.13",
+    "@babel/core": "7.14.5",
+    "@babel/plugin-proposal-class-properties": "7.14.5",
+    "@babel/plugin-proposal-decorators": "7.14.5",
+    "@babel/plugin-proposal-export-default-from": "7.14.5",
+    "@babel/plugin-proposal-object-rest-spread": "7.14.5",
+    "@babel/plugin-proposal-private-methods": "7.14.5",
     "@babel/plugin-syntax-dynamic-import": "7.8.3",
-    "@babel/plugin-transform-modules-commonjs": "7.12.13",
-    "@babel/preset-env": "7.12.16",
-    "@babel/preset-react": "7.12.13",
+    "@babel/plugin-transform-modules-commonjs": "7.14.5",
+    "@babel/preset-env": "7.14.5",
+    "@babel/preset-react": "7.14.5",
     "@babel/preset-typescript": "7.12.16",
+    "@emotion/cache": "11.4.0",
+    "@emotion/core": "11.0.0",
```

Comment:
Is this needed? 🤔

Reply:
After trying to remove them we still need them for the tests